assets/js/components/notifications/UnsatisfiedScopesAlert.js

```diff
@@ -52,6 +52,15 @@ const MESSAGE_MULTIPLE = 'multiple';
 const MESSAGE_SINGULAR = 'single';
 const MESSAGE_GENERIC = 'generic';
 
+/**
+ * Maps unsatisfied scopes to module names.
+ *
+ * @since n.e.x.t
```

Comment:
I've put this as `n.e.x.t` although these functions were introduced previously, it would be too much trouble than worth to dig deeper for the actual release. :D